android/cpp-adapter.cpp

```diff
@@ -1,17 +1,37 @@
 #include <jni.h>
+#include <fbjni/fbjni.h>
+#include <jsi/jsi.h>
+#include <ReactCommon/CallInvokerHolder.h>
 #include "react-native-quick-sqlite.h"
+#include "logs.h"
+#include <typeinfo>
 
-extern "C" JNIEXPORT void JNICALL
-Java_com_reactnativequicksqlite_SequelModule_initialize(JNIEnv *env, jclass clazz, jlong jsiPtr, jstring docPath)
+struct QuickSQLiteBridge : jni::JavaClass<QuickSQLiteBridge>
```

Comment:
Thanks again for figuring this one out @EduFrazao! How did you know a struct was enough? are you an android dev?